src/Shopsys/ShopBundle/Form/Admin/ProductFormTypeExtension.php

```diff
@@ -30,6 +30,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'choices' => $this->productConditionFacade->getAll(),
             'label' => t('Condition'),
         ]);
+
+        $builder->remove('flags');
```

Comment:
this is awesome though :slightly_smiling_face:  :+1:

Reply:
I like it too :) Generally, I like our admin forms :)